pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -904,6 +904,15 @@ func validateSink(
 	if err != nil {
 		return err
 	}
+	u, err := url.Parse(details.SinkURI)
+	if err != nil {
+		return err
+	}
+
+	if u.Scheme == changefeedbase.SinkSchemeCloudStorageHTTP || u.Scheme == changefeedbase.SinkSchemeCloudStorageHTTPS {
```

Comment:
Do we want to do that?
Are we going to bother people who want http with needles warning?

Would it be better to add file-http(s) scheme and prepare http(s) scheme for deprecation?

@amruss ?